VAMobile/tsconfig.json

```diff
@@ -17,6 +17,7 @@
 		"allowSyntheticDefaultImports": true,
 		"esModuleInterop": true,
 		"resolveJsonModule": true,
+		"skipLibCheck": true,
```

Comment:
why did this line need to be added?

Reply:
Short answer: without it there was a litany of (52) TS linting errors from `yarn lint:ci` within node_modules. 

When I dug into it, it seemed the likely culprit was having duplicate type definitions within packages--many seemed to be related to the design system having different (more up-to-date) versions of the i18next and styled-components libraries, but there was also some react/react-native. At some point in the troubleshooting, we were using directly incompatible i18n functionality, but maybe that is no longer the case having sorted out other issues. I didn't dig fully into the react/react-native linting issues, but my assumption was they related to having to include the expo library or it'd crash because the design system package references it even if it can never be hit when running within the app, but had the benefit of allowing the same code with no modifications to run both within the design system repo and as published npm package.

On the whole, it didn't seem like we should really care about TS problems within node_modules if things still run as expected and we should still get linting failures if the issues were seeping into app-level code.